src/components/AppFrame.vue

```diff
@@ -3,6 +3,7 @@
   <div>
     <!-- nav goes here -->
     <NavBar />
+    <ComponentStatusSnack />
```

Comment:
@vickywane usually Vue component names are not prefixed with `Component`. It becomes redundant and noisy as components add up. Let's rename to `statusSnackbar` to keep it close to the standard meaning of the material design component names.